### PR TITLE
Why choose us component is added and call in index.razor

### LINKED ISSUE #6: Separate WhyChooseUsComponent
Almost same as ItemGridComponent.
Need to create WhyChooseUsCardComponent

### PR DESCRIPTION
**Pull request by zeeztasium**
Added WhyChooseUs component in Components folder

closes #6

### CODE REVIEW

File: webapp/Components/WhyChooseUs.razor [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+                    <div class="col-lg-4 col-md-6 col-12">
+                        <!-- Start Single List -->
+                        <div class="single-list wow fadeInUp" data-wow-delay=".2s">
+                            <i class="lni lni-book"></i>
+                            <h4>Fully Documented</h4>
+                            <p>Buy and sell everything from used cars to mobile phones and computer or search
+                                for property.</p>
+                        </div>
+                        <!-- Start Single List -->
+                    </div>
```
Comment (wahidustoz) [inline webapp/Components/WhyChooseUs.razor]: Shularni hammasini WhyChooseUsCardComponent.razor ga oling va dynamically foreach bilan render qiling

File: webapp/Components/WhyChooseUs.razor [hunk 2]
```diff
@@ -0,0 +1,79 @@
+<div class="container">
```
Comment (wahidustoz) [inline webapp/Components/WhyChooseUs.razor]: File nomida *Component so'zi bo'lishi kerak. WhyChooseUsComponent.razor

**Review by wahidustoz — CHANGES_REQUESTED** [review]

**Comment by zeeztasium** [comment]
@wahid-d aytganizdek qildim, tog'rimi?

File: webapp/Components/WhyChooseUsCardComponent.razor [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        <div class="single-list wow fadeInUp" data-wow-delay=".2s">
+        <i class="lni lni-book"></i>
+        <h4>@component.Title</h4>
+        <p>@component.Description</p>
+    </div>
+    }
+</div>
+
+@code {
+    public List<WhyChooseUsCardComponentModel> CardComponents = new List<WhyChooseUsCardComponentModel>()
```
Comment (wahidustoz) [inline webapp/Components/WhyChooseUsCardComponent.razor]: Proporty nomini CardData deb nomlang

File: webapp/Components/WhyChooseUs.razor [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
```
Comment (wahidustoz) [inline webapp/Components/WhyChooseUs.razor]: Yana file nomini o'zgartirmapsizku. `WhyChooseUsComponent.razor`ga

**Review by wahidustoz — CHANGES_REQUESTED** [review]

**Comment by zeeztasium** [comment]
Hammasi done @wahid-d ustoz, build , run ham qildim hammasi ishladi + UI, MERGE qivorin ustoz)))😁

**Pull request merged**